asynchronous-authorization/langchain-next-js/.gitignore

```diff
@@ -1,6 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
+./package-lock.json
```

Comment:
Why was this file added to .gitignore (but the `package-lock.json` file is still under version control)?